build-system/global-configs/canary-config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "allow-doc-opt-in": ["visibility-v3", "amp-animation", "ampdoc-shell"],
+  "allow-doc-opt-in": ["visibility-v3", "amp-animation", "ampdoc-shell", "amp-ima-video"],
```

Comment:
I notice that `visibility-v3` and `amp-animation` have entries a few lines down. Does the amp-ima-video need a value below too?

Reply:
no, those entries can fully launch (e.g. =`1`) or randomly enable (e.g. = `0.4`) am experiment. We want `ima` to stay unlaunched and only allow publisher to opt-in to use it for now.